addons/survey/views/survey_templates_user_input_session.xml

```diff
@@ -153,20 +153,22 @@
                         </div>
                     </div>
                 </div>
-                <div class="o_survey_session_leaderboard w-100 flex-column flex-grow-1" style="display: none;">
+                <div class="o_survey_session_leaderboard w-100 flex-column flex-grow-1" style="display: none; min-width: 75vw; max-height: 80vh;">
```

Comment:
tldr; If you can fix everything in SCSS instead it would be bette. If you can't, please don't forget to clean this in the master port if still applicable.

Details: CSS-based solutions can avoid the need for upgrading the app, and are also preferred to inline `style` attributes (contrary to bootstrap classes).
We can argue though that we would want to fix all or nothing and not have a hybrid fix with updated CSS fixing some things but not the template itself, so in that sense it's OK as you did.